addons/mail/tests/test_link_preview.py

```diff
@@ -88,3 +93,14 @@ def test_02_link_preview_create(self):
                     } for link_preview in message.link_preview_ids]
                 }]
             )
+
+    def test_03_link_preview_create(self):
+        with patch.object(requests.Session, 'head', _patched_head_no_content_type):
```

Comment:
I would patch both even if we don't call `get`, just in case it gets called somehow so it's at least consistent.
As we can't know exactly why it doesn't crash (is it because of the fix or because the header is now valid somehow?)